tools/devops/automation/templates/governance/apiscan.yml

```diff
@@ -27,12 +27,27 @@ steps:
     repositoryAlias: ${{ parameters.repositoryAlias }}
     commit: ${{ parameters.commit }}
 
-- task: DownloadPipelineArtifact@2
-  displayName: Download artifacts
-  inputs:
-    artifactName: not-signed-package
-    allowFailedBuilds: true
-    path: $(Build.SourcesDirectory)/not-signed-package
+- ${{ if or(contains(variables['Build.Reason'], 'ResourceTrigger'), contains(variables['Build.Reason'], 'BuildCompletion')) }}:
+  - download: macios
+    displayName: Download artifacts
+    artifact: not-signed-package
+
+  - pwsh: |
+      $source = "$(Pipeline.Workspace)/macios/not-signed-package"
+      $destination = "$(Build.SourcesDirectory)/not-signed-package"
+      New-Item -ItemType Directory -Force -Path $destination 
+      Write-Host "Moving content from $source to $destination"
+      # move all the files from the source to the destination
+      Get-ChildItem -Path $source -Recurse -File | Move-Item -Destination $destination 
```

Comment:
I don't think this works if there are files in subdirectories.

Can't you just move/copy `$source` to `$destination`?

Reply:
The script is not copying the files as they are, it is flattening the directory and copying all the files in it. So it is nota move or a copy. Is a flattening and a copy.

Reply:
Why would that be needed here and not in the `${{ else }}` section below?

Can you add a comment explaining why flattening is needed?

Reply:
yes, the issue is that the download task downloads under ${pipe-reference}/artifact while the other task does download to the desired path. So we need to do that. I'll add a comment.

Reply:
You can find a successful run here: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=10369133&view=results